src/scene/container/ViewContainer.ts

```diff
@@ -0,0 +1,97 @@
+import { Bounds } from './bounds/Bounds';
+import { Container } from './Container';
+
+import type { PointData } from '../../maths/point/PointData';
+import type { View } from '../../rendering/renderers/shared/view/View';
+import type { BoundsData } from './bounds/Bounds';
+
+export class ViewContainer extends Container implements View
```

Comment:
this class should be `abstract` since we wouldn't want someone to make one of these right?
it also means you can mark `addBounds` and `updateBounds` as abstract